package.json

```
@@ -37,7 +37,7 @@
     "joi": "14.0.4",
     "lodash": "4.17.11",
     "semver": "5.6.0",
-    "sequelize": "4.41.1"
+    "sequelize": "^5.21.5"
```

```suggestion
    "sequelize": "5.21.5"
```
Notice the other libraries are specific, not variable.

---

Updated, also all tests are passing.